e2e/templates/server/deployment.yaml

```diff
@@ -14,16 +14,27 @@ spec:
     metadata:
       labels:
         k8s-app: konnectivity-server
+    spec:
+      priorityClassName: system-cluster-critical
+      tolerations:
+      - key: "CriticalAddonsOnly"
+        operator: "Exists"
+      - operator: "Exists"
+      nodeSelector:
+        node-role.kubernetes.io/control-plane: ""
       containers:
       - name: konnectivity-server-container
         image: {{ .Image }}
+        securityContext:
+          allowPrivilegeEscalation: false
+          runAsUser: 0
```

Comment:
Ditto

Reply:
This error occurs if the server doesn't run as root:

```
│ E0823 21:30:49.548374       1 main.go:48] error: failed to load kubernetes client config: error loading config file "/etc/kubernetes/admin.conf": open /etc/kubernetes/admin.conf: permission denied            │
```